internal/pkg/install/install.go

```diff
@@ -30,11 +30,19 @@ func Install(args string, data *userdata.UserData) (err error) {
 		return nil
 	}
 
+	// Install the bootloader.
+
+	if err = syslinux.Prepare(data.Install.Boot.Device); err != nil {
```

Comment:
This was in the prepare step, but I feel its more of an install step.

Reply:
The prepare step seemed to be all the bits that handled the disk layout/management whereas install was handling the actual filesystem layout. 
I'm mostly indifferent on where it lands, but it could be beneficial to move it here so as we start to think through upgrades we'll be able to handle that here.